src/components/include/utils/make_shared.h

```diff
@@ -124,6 +124,25 @@ SharedPtr<T> MakeShared(
   return Initialize(t);
 }
 
+template <typename T,
```

Comment:
@LuxoftAKutsan May be it is worth now to use `variadic` instead of extending number of parameters every time?

Reply:
@AByzhynar sure, but not as part of this feature.